where_is_my_sddm_theme/Main.qml

```diff
@@ -259,6 +259,13 @@ Rectangle {
                         }
                     })()
 
+				SequentialAnimation on color {
+				        loops: Animation.Infinite
+				        ColorAnimation { from: "#000000"; to: "#FFFFFF"; duration: 350 }
```

Comment:
1. use `config.passwordCursorColor` instead of `"#FFFFFF"`
2. use `"transparent"` instead of "#000000" (because background may be not black)

Reply:
alright, i'll update the PR when i've done it